ghost/admin/app/utils/merge-stats-by-date.js

```diff
@@ -0,0 +1,26 @@
+export default function mergeDates(list) {
```

Comment:
We may want to name this `mergeStatsByDate` too?

Reply:
Yep! Fixed